src/FrontendAccountCreation.Web/Resources/Views/ApprovedPerson/SoleTraderTeamMemberDetails.cy.resx

```diff
@@ -151,7 +151,7 @@
     <value>Ffôn</value>
   </data>
   <data name="TeamMemberDetails.Title" xml:space="preserve">
-    <value>[Welsh] Team Member Details</value>
+    <value>[Welsh] What are their details?</value>
```

Comment:
Why we are touching soletrader We are not doing anything related to sole trader Is the Bug raised for sole trader?